dask/dataframe/io/parquet.py

```diff
@@ -298,7 +298,7 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
 def to_parquet(path, df, compression=None, write_index=None, has_nulls=True,
                fixed_text=None, object_encoding=None, storage_options=None,
                append=False, ignore_divisions=False, partition_on=None,
-               compute=True):
+               compute=True, times='int64'):
```

Comment:
Are there other keyword arguments that we should use here?  Would it make sense to pass through kwargs?

Reply:
`has_nulls=True, write_index=None, partition_on=[], fixed_text=None, append=False, object_encoding='infer', times='int64' in fastparquet.write (aside from other pars that are determined in the dask case. Obviously, this grew organically. It would probably be wrong to refer people to the fastparquet docs here; plus I don't know what will be when arrow writing is plumbed in.